packages/playground/src/components/webhooks/create-webhook-form-data.ts

```diff
@@ -0,0 +1,23 @@
+import { WebhookEvent } from './types'
+
+export const branchEvents: WebhookEvent[] = [
+  { id: 1, event: 'Branch created' },
+  { id: 2, event: 'Branch updated' },
+  { id: 3, event: 'Branch deleted' }
+]
+
+export const tagEvents: WebhookEvent[] = [
+  { id: 4, event: 'Tag created' },
+  { id: 5, event: 'Tag updated' },
+  { id: 6, event: 'Tag deleted' }
+]
+
+export const prEvents: WebhookEvent[] = [
```

Comment:
can we reuse the enums from the types file instead of duplicating all the strings?

Reply:
Already fixed in followup PR